pkg/operator/contenthash/tolerations_test.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 PlanetScale Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package contenthash
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/pointer"
+)
+
+func TestTolerationSecondsNil(t *testing.T) {
```

Comment:
I don't want to block on this, but it would be nice if we also directly test the invariant that caused you to create this PR. The variant being testing that the operator does not remove default tolerances.

Reply:
Good idea. I added a test in the update package.